### DISCUSSION TITLE
Standard adókód és XML

### DISCUSSION THREAD

**Discussion by Edi1978** [nav-gov-hu/eVAT#13] [Kérdések és válaszok - Gépi interfész] [answered]
A standard adókódok és a bevallási sorok 1-1 kapcsolatban állnak egymással (beleértve a részletező sorokat is)?
Egyértelműen fog egy adókód azonosítani egy áfa bevallási sort?
Ha az ügylet alapján részletező sorokban is kell feltüntetni adatot, akkor az egy másik adókód lesz, vagy ugyanaz mintha csak a "fő" sorban kell lennie?

A taxpointDate a pénzforgalmi áfás esetén mit tartalmazzon? (Nyilván a kiegyenlítés dátuma kell ide, azt értjük)
Ugyanaz a számla többször is szerepelhet-e az analitika tételsoraiban … például pénzforgalmi áfás számla részkiegyenlítései esetén?

Az eÁfa M2M bevallásban szerepeltethető-e egyéb könyvelési bizonylat is? … pl.: havi arányos áfa átvezetés bizonylata időszak utolsó napján?
Arányosításba bevont bizonylat esetén mi kerüljön az XML-be? Különös tekintettel a levonásba helyezett adó összegére? Az arány alapján a számlára vonatkozó levonási összeg csak hónap végén, elszámoláskor derül ki. Időközi (tört időszaki) validálás esetén ilyen információ nem áll rendelkezésre még.

**Comment by NTCA-tax** [comment, marked as answer]
A standard adókódokat úgy fogjuk összeállítani, hogy akár a jelenlegi bevallási sorokat is le lehet belőlük képezni. Amennyiben több bevallássort érint, akkor egy komplexebb adókód lesz.

A taxpointDate az adófizetési kötelezettség beálltát, vagy pedig az adólevonási jog dátumát jelenti. Pénzforgalmi elszámolásnál amennyiben részkiegyenlítés történik, akkor azoknak a fizetési időpontjai lesznek. A pénzforgalmi elszámolásnál így előfordulhat, hogy egy számla többször kerül bele az analitikába.

Az analitikába minden olyan számlát, nyugtát, számviteli bizonylatot stb. szerepeltetni kell, amely érinti az áfabevallást.

A levonási arány csak az áfaanalitika készítésénél derül ki, nem számlaadat. Ugyanakkor itt nem arról beszélünk, hogy a számla könyvelésben történő rögzítésénél már minden adatnak ott kell lennie, mivel a levonási arány az ügyviteli folyamat későbbi szakaszában lesz benne. Az áfaanalitika nem a bizonylatok egyszerű kimutatása, hanem számos döntés és számítás eredménye. És bizony előfordulnak olyan esetek, amikor néhány adat, információ csak a bevalláskészítéskor fog rendelkezésre állni.
